website/docs/migration-guides/yew/from-0.18.0-to-0.19.0.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "From 0.18.0 to 0.19.0"
+---
+
+`Yew 0.19.0` has changed a lot, thus this migration will not cover ALL of the changes.
+
+Instead only the most impacting changes are mentioned and the rest should be picked up by cargo.
+
+## html! requirement for braces around most props
+
+Put it simply almost all the time you have to provide braces for props:
+
+```rust {4}, ignore
+let super_age = 1;
+html!{
+    <JapaneseYew
+        age=super_age // ! Will throw an error
+    >
+}
+```
+
+```rust {4}, ignore
+let super_age = 1;
+html!{
+    <JapaneseYew
+        age={super_age} // Correct
+    >
+}
+```
+
+Also now you can use a shorthand if prop and variable names are the same:
+
+```rust {4}, ignore
+let age = 1;
+html!{
+    <JapaneseYew
+        {age}
+    >
+}
+```
```

Comment:
Seems like a good place to use [tabs](https://docusaurus.io/docs/markdown-features/tabs)

Reply:
I cant seem to make them work, any pointers?

Reply:
I have ruined my work env while trying to make this work and i am giving up and will not use tabs here

Reply:
@hamza1311 Figured it out, both workspace and tabs. Just needed a version bump for the latter. But I will do this change in a separate PR.

Reply:
added tabs as when #2215 gets merged, they will work